configs/tsconfig.defaults.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compileOnSave": false,
+  "compilerOptions": {
+    "target": "es5",
+    "module": "es2015",
+    "moduleResolution": "node",
+    "outDir": "dist",
+    "importHelpers": true,

```

Comment:
I have not added `tslib` as a peerDependency, though `rxjs` has it as a dependency and since `rxjs` is a peerDependency it should still work. Though, it might still be good to be explicit and add it?

Reply:
I added `tslib` as a peerDependency. I used the same version as RxJS for now.